### ISSUE TITLE
Marking all (bright) tiles from 20240126 as unsure

### ISSUE THREAD

**Issue by forero** [desihub/desisurveyops#176]
While running QA here https://data.desi.lbl.gov/desi/spectro/redux/daily/nightqa/20240126/tileqa-20240126.pdf I noticed that all bright tiles triggered an Alert VCCDSEC on all the B cameras. The sframesky also look unusual, with sharp unmasked negative features in the B cameras and overall noise across all cameras and petals. They resemble what I would expect from a backup tile. 

<img width="951" alt="Screen Shot 2024-01-27 at 3 38 09 PM" src="https://github.com/desihub/desisurveyops/assets/1848618/0978f884-aaa5-4ebe-8da3-09bb5674071d">

Furthermore, some tiles triggered a ZSKYCHI2PDF warning, and their redshift distributions seem to have a gap around z=0.25.

<img width="1192" alt="Screen Shot 2024-01-27 at 3 38 56 PM" src="https://github.com/desihub/desisurveyops/assets/1848618/f6f6496c-a8cd-48f1-a694-1a079114d4fa">

I marked all tiles as unsure.

**Comment by schlafly** [comment]
I think these are fine.  Re VCCDSEC, yes, they did some work on the B cameras.  But the B cameras aren't affected by the same instability as the R and Z cameras and I would usually ignore this for the B cameras.  The negative features around wavelength index 800 or so on some petals are related to variation in humidity and the collimator coating; the level of dip I see there looks modest to me.  These used to be much worse before Julien's corrections; see e.g. https://data.desi.lbl.gov/desi/spectro/redux/daily/nightqa/20220101/sframesky-20220101.pdf for a really old example.

The SKYCHI2PDF warnings are a good reason to be concerned.  Coupled with the feature around 0.25 in the redshift distributions I can see marking those as unsure.  However I can't think of any times we've noticed a deficit of redshifts as a problem; it's usually easier to spot defects in the data attracting redshifts than it is to spot where they've drawn redshifts from.  So I suspect this is just a fluctuation and we could mark good.

The nominal n(z)/n_ref(z) is pretty good, and the petal n(z) also look pretty nominal to me; I think we'd spot issues there if we needed to worry.  @ashleyjross , I'm going to recommend we set these as good, but if you have a quick diagnostic to assess those n(z) that is better than the nominal n(z) / n_ref(z), that could be useful.
